src/app/nation/nation.component.ts

```diff
@@ -124,13 +119,8 @@ export class NationComponent implements OnInit, AfterViewInit {
           formdata[ '_id' ] = data.id;
           formdata[ '_rev' ] = data.rev;
           this.nations.data.push(formdata);
-          console.log(this.nations.data);
+          this.nations.data = this.nations.data;
```

Comment:
i think it was changed with some other event in one of comment. So why is this here again?

Reply:
Raju : i kept this code in my pr just to test  sorry i forget to remover it before creating PR